src/components/TokenNFTPreview.js

```diff
@@ -21,26 +23,49 @@ const TokenNFTPreview = (props) => {
     return null;
   }
 
-  const onPLayVideo = () => {
+  const onPLayMedia = () => {
```

Comment:
This method name should be `onPlayMedia`, right? I think the L was not supposed to be uppercase

Reply:
Did it work during tests?

Reply:
Probably yes, the execution call is also using L. It's just to keep the pattern.

Reply:
Ok, my fingers betrayed me in conspiracy with code completion, haha. Fixed.

Reply:
![image](https://user-images.githubusercontent.com/449119/122590389-5773d900-d016-11eb-89ee-c95d7e93fd3e.png)